client/src/LandingPages/SignupPage/SignupPage.jsx

```diff
@@ -3,62 +3,133 @@ import './SignupPage.css';
 import { Link } from 'react-router-dom';
 import LandingPageNav from '../components/LandingPageHeader/LandingPageNav';
 import { useState } from 'react';
+import FormInput from '../components/Forms/FormInput'
+import PasswordValidation from '../components/Forms/PasswordValidation';
+import { MdAlternateEmail } from "react-icons/md";
+import { TbPencilCheck } from "react-icons/tb";
+import { RiLockPasswordFill } from "react-icons/ri";
+import { FaUserPlus } from "react-icons/fa6";
+import * as authService from '../../services/authService';
 
-const SignupPage = () => {
+const SignupPage = ({ setUser }) => {
   const [inputValue, setInputValue] = useState({
+    firstName: '',
+    lastName: '',
     email: '',
-    password: ''
+    password: '',
+    confirmPassword: '',
   });
 
+  const [errorMsg, setErrorMsg] = useState('')
+  const [successMsg, setSuccessMsg] = useState('')
+
```

Comment:
Good job with ensuring that users who input duplicate emails aren't being created! However, there needs to be some error message that alerts the user that signup failed and that they should try again. I can see that you have state set up for error messages and success messages, but you didn’t display the message anywhere on the page.